src/main/java/seedu/travelr/model/ModelManager.java

```diff
@@ -172,7 +187,7 @@ public void updateFilteredTripList(Predicate<Trip> predicate) {
     @Override
     public void updateFilteredEventList(Predicate<Event> predicate) {
```

Comment:
Should we rename this to updateAllEventList? If i'm not wrong FilteredEventList still refers to BucketList

Reply:
Sure, that's a good idea.